customizations/services.py

```diff
@@ -0,0 +1,192 @@
+from django.db import transaction, DatabaseError
+from django.shortcuts import (
+    get_object_or_404,
+)
+from customizations.utils import (
+    upload_file,
+    get_token,
+    create_webhook,
+    upload_file_dropbox,
+    get_unique_file_name,
+)
+from customizations.models import (
+    Customization,
+    TicketTemplate,
+    CustomEmail,
+    UserWebhook,
+    BaseTicketTemplate,
+)
+import re
+import cStringIO
+from PIL import Image
+from django.core.files.base import ContentFile
+
+
+def get_image(request_image_data, user):
+    img_io = cStringIO.StringIO()
+    image_base = request_image_data
+
+    image_data = re.sub(
+        '^data:image/.+;base64,', '', image_base
+    ).decode('base64')
+    image = Image.open(cStringIO.StringIO(image_data))
+
+    image.save(img_io, format=image.format, quality=100)
```

Comment:
At the end this will save the image using base64, maybe you could clarify this process, using a better method name and documenting it.